msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.props

```diff
@@ -12,6 +12,8 @@ Copyright (C) 2013-2014 Xamarin. All rights reserved.
 ***********************************************************************************************
 -->
 
+<!-- Unlike Xamarin.Mac.Common.targets, which is not not shared by the ObjC Bindings targets (yet), 
```

Comment:
Minor: "not not shared", so it is shared? :P

Reply:
Good catch.